fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -505,7 +708,17 @@ def cellpose_segmentation(
             )
             channels = [0, 0]
 
-        fov_mask = segment_FOV(
+        img_np, background_3D, current_label = preprocess_cellpose_input(
+            image_array=img_np,
+            use_masks=use_masks,
+            region=region,
+            # primary_label_path=f"{zarrurl}labels/{primary_label_name}/0",  # FIXME: which level? # noqa
+            current_label_path=f"{zarrurl}labels/{output_label_name}/0",  # FIXME: which level? # noqa
```

Comment:
We always want to create output at level 0 first, right?

Reply:
Agreed

Reply:
Agreed. This is the level we are writing, so there is no ambiguity.